src/stream.ts

```diff
@@ -32,8 +33,8 @@ export class WebRTCStream implements Stream {
   metadata: Record<string, any>;
   private readonly channel: RTCDataChannel;
 
-  source: Source<Uint8Array> = pushable();
-  sink: Sink<Uint8Array, Promise<void>>;
+  source: Source<Uint8ArrayList> = pushable();
+  sink: Sink<Uint8ArrayList | Uint8Array, Promise<void>>;
```

Comment:
@ckousik - please look at these changes, to be in compliance with the latest Stream interface

````
src/stream.ts:36:3 - error TS2416: Property 'sink' in type 'WebRTCStream' is not assignable to the same property in base type 'Stream'.
  Type 'Sink<Uint8Array, Promise<void>>' is not assignable to type 'Sink<Uint8ArrayList | Uint8Array, Promise<void>>'.
    Type 'Uint8ArrayList | Uint8Array' is not assignable to type 'Uint8Array'.
      Type 'Uint8ArrayList' is missing the following properties from type 'Uint8Array': BYTES_PER_ELEMENT, buffer, byteOffset, copyWithin, and 21 more.

36   sink: Sink<Uint8Array, Promise<void>>;
````